server/lib/tuist/projects/vcs_connection.ex

```diff
@@ -0,0 +1,42 @@
+defmodule Tuist.Projects.VCSConnection do
+  @moduledoc """
+  A module that represents VCS connections between Tuist projects and external repositories.
+  """
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  alias Tuist.Accounts.Account
+  alias Tuist.GitHubAppInstallations.GitHubAppInstallation
+  alias Tuist.Projects.Project
+
+  @primary_key {:id, UUIDv7, autogenerate: true}
+  @foreign_key_type UUIDv7
+  schema "vcs_connections" do
+    field :provider, Ecto.Enum, values: [github: 0]
+    field :repository_full_handle, :string
+
+    belongs_to :project, Project, type: :integer
+    belongs_to :created_by, Account, foreign_key: :created_by_id, type: :integer
```

Comment:
Why `Account`? Aren't users the only ones that can create these connections?

Reply:
thought about it – and conceptually it's better to refer to a user here after all. will change this